### ISSUE TITLE
The documentation does not include imports.

### ISSUE THREAD

**Issue by skorokithakis** [beerfactory/hbmqtt#50]
It's very hard to follow the documentation because the examples don't show where the imports come from. To run an example, I have to look through the docs and see where each name is imported from. Please add imports to the doc examples.

**Comment by njouanin** [comment]
Note that complete examples are also available in the [samples](https://github.com/beerfactory/hbmqtt/tree/develop/samples) directory

**Comment by skorokithakis** [comment]
Ah, that's helpful, thank you. Still, it would be helpful if the includes were shown in the docs.

**Comment by njouanin** [comment]
Done
